docs/platform/block-storage/block-storage-use-cases/index.md

```diff
@@ -8,58 +8,80 @@ license: '[CC BY-ND 4.0](https://creativecommons.org/licenses/by-nd/4.0)'
+There are a few media library applications, most notably [Plex](https://www.plex.tv/), that offer media streaming functionality to internet enabled devices. The media libraries these applications serve can quickly grow in size, depending on the number of movie and audio files they contain. Using a Block Storage Volume can provide you with storage capable of growing with the needs of your library.
 
 {{< note >}}
-For more information on using Plex with a block storage volume, see our [Using a Block Storage Volume with Plex](/docs/platform/block-storage/how-to-use-block-storage-with-your-linode/#how-to-detach-a-block-storage-volume-from-a-linode) guide.
+For more information on using Plex with a Block Storage Volume, see our [Using a Block Storage Volume with Plex](/docs/platform/block-storage/how-to-use-block-storage-with-your-linode/#how-to-detach-a-block-storage-volume-from-a-linode) guide.
 {{< /note >}}
 
 ### Ephemeral Storage
 
-Various stages of the software development lifecycle can create large amounts of temporary data, such as buffers, builds, and cache and session data. While this data might only exist for a short period of time, it requires and utilizes storage space. Creating a Block Storage Volume just for ephemeral data is a good use case for times when the storage supplied with your server is not  on it's own,   or for when you need extra space for a short period of time.
+Various stages of the software development lifecycle can create large amounts of temporary data, such as buffers, builds, and cache and session data. While this data might only exist for a short period of time, it requires and utilizes storage space. Creating a Block Storage Volume just for ephemeral data is a good use case for times when the storage supplied with your instance is not enough, or for when you need extra space for a short period of time.
 
 ### Data Backups
 
-Having backups of your data is always a good idea, and Block Storage Volumes make for scalable and quickly accessible backup mediums. Store anything that you might need to quickly transfer to another server, or anything that you might need at a moment's notice.
+Having backups of your data is always a good idea, and Block Storage Volumes make for scalable and quickly accessible backup mediums. Store anything that you might need to quickly transfer to another instance, or anything that you might need at a moment's notice.
 
 ### Boot Disks
 
-You can boot from disk images installed to a Block Storage Volume. This provides a cost effective means of maintaining an image that can be attached to a new Linode if for some reason you need to restore a failed server.
```

Comment:
The second sentence of this seemed like it was compressing two different ideas:

- That a bootable Volume can be a money-saver
- That it can help with recovery efforts

So I expanded on each and split them into several sentences.